FINANCES.md

```diff
@@ -1,20 +1,20 @@
 # Finances
 
-The Tessel Project (as of May 2016) has its funds held by the non-profit Dojo Foundation.
+The Tessel Project (as of May 2016) has its funds held by non-profit foundations. The funds are currently held by the Bocoup Foundation.
```

Comment:
Are the funds being help by multiple foundations? Or just the Bocoup Foundation now?

> The Tessel Project (as of May 2016) has its funds held by non-profit **foundations.**